### ISSUE TITLE
SELECT clause from mapped classes

### ISSUE THREAD

**Issue by raderio** [PicnicSupermarket/jolo#3]
What do you think about this feature 

```
List<MyPojo> list4 = dsl.select(MyPojo.class).from(T).fetch();
```

https://github.com/jOOQ/jOOQ/issues/9126
https://github.com/jOOQ/jOOQ/issues/9005

It have a lot in common with Jolo.

**Comment by Mordavolt** [comment]
Hey @raderio, thanks for reaching out.

This looks like a feature that could potentially do some of the things Jolo does. I'd argue that https://github.com/jOOQ/jOOQ/issues/9288 looks even more like Jolo.

That being said, and you can see some great points here https://github.com/jOOQ/jOOQ/issues/9288#issuecomment-535832929, Jooq will likely never fully implement what Jolo does. Jooq is meant to retrieve data the way SQL engine thinks about it, as abstract tuples. We've built Jolo to complement that and in turn map data to a graph of domain objects, which is an OOP way of looking at it.

To sum it up, thanks for the heads up, it would be indeed a nice feature in Jooq, but unlikely that it would be sufficient for us to forgo Jolo. What do you think about it?

**Comment by Mordavolt** [comment]
Closing since there is nothing to do here.
